scripts/generate-integration-pages.ts

```diff
@@ -178,6 +179,47 @@ function absoluteLinks({ base }: { base: string }) {
 	};
 }
 
+/** Remark plugin to replace GitHub . */
+function replaceAsides() {
+	return function transform(tree: Root) {
+		// visit blockquote
+		visit(tree, 'blockquote', (node) => {
+			// first child (<blockquote><p>...</p></blockquote>)
+			const [firstChild, trailingText] = node.children[0].children;
+
+			if (firstChild.type === 'strong') {
+				if (
+					// check for **Note** or **Warning**
+					firstChild.children[0].value.includes('Note') ||
+					firstChild.children[0].value.includes('Warning')
+				) {
+					const AsideType =
+						firstChild.children[0].value.toLowerCase() === 'note' ? 'note' : 'caution';
```

Comment:
This technically makes “caution” our default note type. Not sure we’d ever actually hit this, but just in case I think defaulting to “note” probably makes more sense:

```suggestion
						firstChild.children[0].value.toLowerCase() === 'warning' ? 'caution' : 'note';
```

Reply:
caution this may explode CI 😆🔥😇